app/views/offices/_form.html.slim

```diff
@@ -27,6 +27,6 @@
             .columns.small-12.end
               = b.label do
                 = b.check_box
-                = b.text
+                = "#{b.text} (#{@becs.where(jurisdiction_id: b.value).first.code})"
```

Comment:
> I think we should avoid doing sql queries in the views at any cost.

Fully agree.

> (not sure exactly how)

The same here, hence the 'SQL in the views'.

If you can come up with a good clean way of doing it, I'd love to know.

The reason I left this filth in, is because I couldn't figure out a clean solution and figured that this is such a small query that it can be left in. But since I don't know of a cleaner way, I'd love to hear all the suggestions of doing it better.